R/summarize_draws.R

```diff
@@ -0,0 +1,108 @@
+#' @title Calculate summary statistics
```

Comment:
Maybe the filename should be "summarize" to be more generic?

Reply:
Yeah good catch